src/items/itemsTable.component.tsx

```diff
@@ -280,9 +285,20 @@ export function ItemsTable(props: ItemTableProps) {
             onClose={() => {
               table.setCreatingRow(null);
             }}
-            type="add"
+            type={itemDialogType}
             catalogueCategory={catalogueCategory}
             catalogueItem={catalogueItem}
+            selectedItem={
+              itemDialogType === 'create'
+                ? undefined
+                : {
+                    ...row.original,
+                    notes:
+                      itemDialogType === 'save as'
+                        ? `${row.original.notes}.\nThis is a copy of the item with this ID: ${row.original.id}`
```

Comment:
```suggestion
                        ? `${row.original.notes || ''}\n\nThis is a copy of the item with this ID: ${row.original.id}`
```
I'm not keen on adding `.` the prior notes could already have one/may not be full sentences. Also if the notes are currently null, it produces
![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/542a22ae-3377-48ec-bcc0-2d13893ccfd5)

Could a test be added to cover this?

Reply:
This line is covered in the next pr for the edit

Reply:
Where was this covered? It still looks like there is only one test for it, I get that the coverage says it is but that just means its rendered somewhere but there doesn't seem to be anything checking the actual message so it could catch this in the future. Happy for it to be addressed in edit though.

Reply:
when it is just `{row.original.notes} ` that is going to be tested in the edit automatically. As the edit using just notes with no extra text

Reply:
Yes but I meant when `row.orignal.notes === null` which is what caused this bug in the first place

Reply:
Done